queries/capnp/highlights.scm

```diff
@@ -0,0 +1,118 @@
+; Preproc (?)
+
+(unique_id) @preproc
+
+; Includes
+
+[
+  "import"
+  "$import"
+] @include
+
+(import_path) @string
+
+; Types
+
+(primitive_type) @type.builtin
+
+[
+  "annotation"
+  "enum"
+  "group"
+  "interface"
+  "List"
+  "struct"
+  "union"
+] @type.builtin
+
+; Methods
+
+(method_identifier) @method
+
+; Fields
+
+(field_identifier) @field
+
+; Parameters
+
+(param_identifier) @parameter
+(return_identifier) @parameter
+
+; Variables
+
+(identifier) @variable
+
+; Constants
+
+(const_identifier) @constant
+(enum_member) @constant
+
+; Types
+
+(enum_identifier) @type
+(extend_type) @type
+(field_type) @type
+(generic_identifier) @type
+(type_identifier) @type
+
+; Attributes
+
+(attribute) @attribute
+(annotation_identifier) @attribute
+
+; Operators
+
+[
+ ; @ ! -
+  "="
+] @operator
+
+; Keywords
+
+[
+  "const"
+  "extends"
+  "import"
+  "$import"
+  "namespace"
+  "using"
+] @keyword
```

Comment:
Isn't `include` already `@include`

Reply:
Not sure what you mean? Do you mean the "import" and "$import" being a duplicate kind of?

Reply:
I wanted to say that "include" shouldn't be `@keyword` when it's already `@include`

Reply:
fixed